lib/v2/screens/authentication/recover/recover_account_search/recover_account_screen.dart

```diff
@@ -91,6 +91,8 @@ class _RecoverAccountScreenState extends State<RecoverAccountScreen> {
                             imageUrl: state.accountImage,
                             account: state.userName!,
                             name: state.accountName,
+                            resultCallBack: () =>
```

Comment:
this is from the previous PR

Reply:
you can always checkout master, then branch from master to avoid this.

Reply:
true...